dev/dependencyList

```diff
@@ -87,9 +101,15 @@ metrics-json/4.2.8//metrics-json-4.2.8.jar
 metrics-jvm/4.2.8//metrics-jvm-4.2.8.jar
 netty-all/4.1.73.Final//netty-all-4.1.73.Final.jar
 netty-buffer/4.1.73.Final//netty-buffer-4.1.73.Final.jar
+netty-codec-dns/4.1.74.Final//netty-codec-dns-4.1.74.Final.jar
```

Comment:
gRPC uses a shaded netty, why do we need those?

Reply:
> gRPC uses a shaded netty, why do we need those netty stuff?

These netty jars are a dependency of jetcd, had try to use netty-shaded, but got `ClassNoDefException`. Classes in  `netty-shaded` have prefix `io.grpc`, jetcd can not recognized. Any suggestions?

Reply:
https://github.com/etcd-io/jetcd/pull/1068 seems going to provide an all-in-one client jar then we can get rid of those dependencies.